docs/best_practices/time_series.rst

```diff
@@ -17,76 +19,125 @@ Keep in mind that the dimensions are reversed in MatNWB, so in memory in MatNWB
+brain that are monitoring neural activity, and only those electrodes should be in the electrodes table.
+
+For non-neural electrodes that still may store and report raw values in Volts, simply use a general ``TimeSeries``
+object with ``unit`` set to ``Volts``.
+
 
 
-.. _best_practice_time_series_break_in_continuity
+.. _best_practice_timestamps_ascending:
 
 Breaks in Continuity
 ~~~~~~~~~~~~~~~~~~~~
-TimeSeries data should be stored as one continuous stream.
+TimeSeries data should generally be stored as one continuous stream as it was acquired, not by trial as is often
+reshaped for analysis.
 
-Data should be stored in one continuous stream, as it is acquired, not by trial as is often reshaped for analysis.
+Data can be trial-aligned on-the-fly using the ``trials`` table.
 
-Data can be trial-aligned on-the-fly using the trials table.
+Storing measured data as a continuous stream ensures that other users have access to the inter-trial data, and that we
+can align the data within any specifiable window.
 
-Storing measured data as a continuous stream ensures that other users have access to the inter-trial data, and that we can align the data with whatever window they need.
+If you only have data spanning specific segments of time, then only include those timepoints in the data, see
+:ref:`best_practice_regular_timestamps` for more information.
 
-If you only have data spanning specific segments of time, then only include those timepoints in the data.
+A primary implication is that the values in ``timestamps``, as well as the corresponding ordering of their indices
+in the ``data`` array, should always be strictly increasing.
+
+Check function: :py:meth:`~nwbinspector.checks.time_series.check_timestamps_ascending`
 
-One could use timestamps for this, even if there is a constant sampling rate within each segment, and have the timestamps correctly reflect the gaps in the recording.
 
 
 .. _best_practice_regular_timestamps:
 
 Timestamps vs. Start & Rate
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use timestamps, even if there is a constant sampling rate within each segment, and have the timestamps correctly
-reflect the gaps in the recording. Use the TimeSeries.description field to explain how the data was segmented.
-If the sampling rate is constant, use rate. TimeSeries allows you to specify time using either timestamps or rate and starting_time (which defaults to 0).
-For TimeSeries objects that have a constant sampling rate, rate should be used instead of timestamps. This will ensure that you can use analysis and
-visualization tools that rely on a constant sampling rate.
+``TimeSeries`` allows you to specify time using either ``timestamps`` or ``rate`` together with ``starting_time``
+(which defaults to 0).
+
+If the sampling rate is constant, then specify the ``rate`` and ``starting_time`` instead of writing the full ``timestamps`` vector.
```

Comment:
```suggestion
(which defaults to 0). If the sampling rate is constant, then specify the ``rate`` and ``starting_time`` instead of writing the full ``timestamps`` vector.
```